OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -51,6 +52,15 @@ public EditorDefaultBrush(EditorViewportControllerWidget editorWidget, WorldRend
 				.ToDictionary(r => r.Info.ResourceType, r => r);
 		}
 
+		long CalculateActorSelectionPriority(EditorActorPreview actor)
+		{
+			var center = new int2(actor.Bounds.X, actor.Bounds.Y);
+			long priority = (center - worldPixel).Length;
```

Comment:
I believe he wants to enforce long over int because of the shifting by 32 bits that is done on the next line.
Interesting approach though. I think this method can use a comment or two.